se.manyver.Manyverse.yaml

```diff
@@ -0,0 +1,63 @@
+app-id: se.manyver.Manyverse
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: manyverse
+separate-locales: false
+rename-desktop-file: manyverse.desktop
+rename-icon: manyverse
+finish-args:
+  - --socket=x11
+  - --socket=wayland
```

Comment:
does it support wayland?

Also even as fallback `--share=ipc` is needed for x11

Reply:
This is a part where, honestly, I am not actually sure. It's an electron app, does it make sense to add the wayland socket for an electron app, I am not sure. I'd appreciate any suggestion there :)

Reply:
in general Electron doesn't support wayland.